docs/MANUAL-TESTING.md

```diff
@@ -0,0 +1,114 @@
+Manual Testing
+==============
+
+This document describes a high-level script of manual tests to check for. We
+should aim to replace items on this list with automated Spectron test cases.
+
+Image Selection
+---------------
+
+- [ ] Cancel image selection dialog
+- [ ] Select an unbootable image (without a partition table), and expect a
+  sensible warning
+- [ ] Attempt to select an unsupported image
+- [ ] Change image selection
+- [ ] Select a Windows image, and expect a sensible warning
+
+Drive Selection
+---------------
+
+- [ ] Open the drive selection modal
+- [ ] Remove all drives
+- [ ] Switch drive selection
+- [ ] Insert a single drive, and expect auto-selection
+- [ ] Insert more than one drive, and don't expect auto-selection
+- [ ] Insert a locked SD Card and expect a warning
+- [ ] Insert a too small drive and expect a warning
+- [ ] Put an image into a removable drive and attempt to flash the image to the
+  drive that contains it
+- [ ] Attempt to flash a compressed image into a drive that is big enough to
+  hold the compressed image, but not big enough to hold the uncompressed
+  version
+- [ ] Enable "Unsafe Mode" and attempt to select a system drive
+- [ ] Enable "Unsafe Mode", and if there is only one system drive (and no
+  removable ones), don't expect autoselection
+
+Image Support
+-------------
+
+Run the following tests with and without validation enabled:
+
+- [ ] Flash an uncompressed image
+- [ ] Flash a Bzip2 image
+- [ ] Flash a XZ image
+- [ ] Flash a ZIP image
+- [ ] Flash a GZ image
+- [ ] Flash a DMG image
+- [ ] Flash an image whose size is not a multiple of 512 bytes
+- [ ] Flash a compressed image whose size is not a multiple of 512 bytes
+- [ ] Flash an archive whose image size is not a multiple of 512 bytes
+- [ ] Flash an archive image containing a logo
+- [ ] Flash an archive image containing a blockmap file
+- [ ] Flash an archive image containing a manifest metadata file
+
+Flashing Process
+----------------
+
+- [ ] Disable validation and flash an image
+- [ ] Unplug the drive during flash or validation
+- [ ] Click "Flash", cancel elevation dialog, and click "Flash" again
+- [ ] **Cancel the application close warning dialog**
+
+### Child Writer
+
+- [ ] **Kill the child writer process (i.e. with `SIGINT` or `SIGKILL`), and
+  check that the UI reacts appropriately**
+- [ ] **Close the application while flashing using the window manager close
+  icon**
+- [ ] **Close the application while flashing using the OS keyboard shortcut**
+- [ ] **Close the application from the terminal using Ctrl-C while flashing**
+- [ ] **Force kill the application (using a process monitor tool, etc)**
+
+In all these cases, the child writer process should not remain alive. Note that
+in some systems you need to open your process monitor tool of choice with extra
+permissions to see the elevated child writer process.
+
+GUI
+----
+
+- [ ] Close application from the terminal using Ctrl-C
```

Comment:
Perhaps this should clarify "...while Etcher is idle" to avoid overlap with the previous section?

Reply:
Done